doc/developer-guides/hld/virtio-net.rst

```diff
@@ -509,6 +509,39 @@ is the virtual NIC created by acrn-dm:
       collisions:0 txqueuelen:1000
       RX bytes:0 (0.0 b) TX bytes:0 (0.0 b)
 
+How to Use MacVTap Interface
+============================
+In addition to Tap interface, ACRN also supports MacVTap interface. 
+MacVTap replaces the combination of the tap and bridge drivers with 
+a single module based on MacVLan driver. With MacVTap each
+virtual network interface is assigned its own MAC, IP address 
+and is directly attached to physical interface of the host machine
+to improve throughput and latencies.
+
+Since MacVLan driver is not enabled by default, ACRN kernel needs to be 
+built with these config options ``CONFIG_MACVLAN=y`` and ``CONFIG_MACVTAP=y``
+to start using MacVTap interface.
```

Comment:
@vijaydhanraj , these options have been enabled by default according the [default SOS kernel config file](https://github.com/projectacrn/acrn-kernel/blob/master/kernel_config_uefi_sos#L2091-L2092). FYI.

Reply:
thanks @ttzeng, I will update this.